Makefile

```diff
@@ -56,3 +56,7 @@ update-yaraforge:
 	mkdir -p out
 	curl -sL -o out/yaraforge.zip https://github.com/YARAHQ/yara-forge/releases/latest/download/yara-forge-rules-full.zip
 	unzip -o -j out/yaraforge.zip packages/full/yara-rules-full.yar -d rules/third_party/
+
+.PHONY: update-threathunting-keywords
+update-threathunting-keywords:
```

Comment:
It really sucks that there are no tags or releases. I see this done in other places too, and I'm wondering if we really should be locking this into a sha, so for example, something like this:
```
https://github.com/mthcht/ThreatHunting-Keywords-yara-rules/blob/a21391e7280a4347dd7faebd7b5f54344b484ec7/yara_rules/all.yara
```

To make sure things don't change from underneath us. Then of course, we'd have to bump these somehow. But, in any case it makes me uncomfortable.

Reply:
Updated in [`682f39c` (#160)](https://github.com/chainguard-dev/bincapz/pull/160/commits/682f39ca7c8c7d7e27b2c06b0f753cf79a794d50) -- let me know what you think about that approach!

Reply:
Hello @egibs @vaikas FYI i put the first release here https://github.com/mthcht/ThreatHunting-Keywords/releases, it will be updated every month or every two months with more details in the next releases